File: src/main/java/com/drumonii/loltrollbuild/model/TrollBuild.java
package com.drumonii.loltrollbuild.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.*;

import java.util.List;

/**
 * Troll Build (random build).
 */
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class TrollBuild {

	@JsonProperty
	@Getter @Setter private List<Item> items;

	@JsonProperty
	@Getter @Setter private List<SummonerSpell> summonerSpells;

	@JsonProperty
	@Getter @Setter private Item trinket;

}

File: frontend/src/app/troll-build/model/troll-build.ts
import { Item } from '@ltb-model/item';
import { SummonerSpell } from '@ltb-model/summoner-spell';

/**
 * Troll Build (random build).
 */
export class TrollBuild {
  items: Item[];
  summonerSpells: SummonerSpell[];
  trinket: Item;
}
